tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -692,4 +693,935 @@ public function provide70Cases()
             ),
         );
     }
+
+    /*
+    |--------------------------------------------------------------------------
+    | Test sorting by length
+    |--------------------------------------------------------------------------
+    */
+
+    /**
+     * @expectedException \PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException
```

Comment:
Please remove all these annotations, and put use the newer syntax please, as updated in the rest of the codebase. See https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2381.